### DISCUSSION TITLE
Github graphical interface for branches with name containing the "/" symbol

### DISCUSSION THREAD

**Discussion by sangallidavide** [community/community#62040] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I think the following is a bug of the github website.
I searched among the discussions and I did not find anything about this.
Not 100% sure I'm posting in the proper place.

In short, when I navigate a "standard" branch of my project, such as "master" this is the aspect
https://github.com/sangallidavide/yambo/tree/master
![image](https://github.com/community/community/assets/22720887/1b76836b-337e-43e9-be45-3013bef99504)

However if I switch to a branch such as "phys/magnons" the website changes aspects
https://github.com/sangallidavide/yambo/tree/phys/magnons
![image](https://github.com/community/community/assets/22720887/53eb0f42-cd71-4c12-b2f8-c5fb7f7415da)

That is, it behaves as if I were exploring a folder inside a branch, while, it should not

**Comment by look** [comment]
Thanks for the report @sangallidavide. I confirmed this only happens when the branch name includes a `/`. I passed the report on to the frontend team.

**Comment by RamtinKosari** [comment]
using "/" in naming your branch makes the words before "/" as an directory, for example in name of "phys/magnons" in .git directory it will creates
a directory named "phys" and inside it creates another directory named "magnons", if you do tree command on linux to show all .git directories in tree format you will get this :
├── heads
│   ├── master
│   └── phys
│       └── magnons
as you see, magnons is sub directory of phys

so you should use other separating symbols like "-" or "_" if you want to see like standard mode
@sangallidavide

**Reply by look** [reply, marked as answer]
This is correct, but it is not intentional that the UI changes when you include a `/` in your branch name. We are going to fix that behavior.

**Comment by sangallidavide** [comment]
I'd say not a big surprise that it is a folder for "tree".
For the "tree" command everything is either a file or a folder, it does not  know anything about branches.

On the other hand "/" is a perfectly legal character for a branch name
https://git-scm.com/docs/git-check-ref-format

As far as I understand, this remains a limitation of the github graphycal interface, not an issue of git
```
$ git branch 
+ bug-fixes
* master
+ phys/magnons
```
```
$ git check-ref-format --branch phys/magnons
phys/magnons
```
```
$ git reflog 
5fecf4d2b (HEAD -> phys/magnons, origin/phys/magnons) HEAD@{0}: reset: moving to HEAD
5fecf4d2b (HEAD -> phys/magnons, origin/phys/magnons) HEAD@{1}: 

```
